### PR TITLE
Merge last version with vignetting and random track position on x-y

### PR DESCRIPTION
**Pull request by SamueleTorelli**

### CODE REVIEW

**Comment by SamueleTorelli** [comment]
last version with vignetting and random track position on x-y

File: MC_data_new.py [hunk 1]
```diff
@@ -166,439 +236,369 @@ def SaveEventInfo(info_dict, folder, out):  # THIS FUNCTION IS CURRENTLY NOT USE
...
-                    if opt.events==-1:
-                        totev=tree.GetEntries()
+                    if opt.events<=tree.GetEntries():
+                        totev=opt.events
                     else:
+                        totev=tree.GetEntries()
+                    
+                VignMap=rt.TH2D()
+                if(opt.Vignetting):
+                    VignFile=rt.TFile.Open('VignettingMap/'+opt.Vig_Map,'read')
```
Comment (pietro14) [inline MC_data_new.py:362]: @SamueleTorelli I am trying to run the code. Where can I find the vignetting map you're using here? Also, I see the parameter `Vig_Map` is only in `ConfigFile_MAN.txt`. So we can remove `ConfigFile_new.txt`, right? Thanks

Comment (SamueleTorelli) [reply MC_data_new.py:362]: Sorry, they are in my repository, they were not added to git. By the way, ConfigFile_MAN.txt is for MANGO, while ConfigFile_new.txt is for lime, you should use the second, adding the vignetting map information. The maps are now on my repository, which I will merge after this commit is accepted

**Comment by pietro14** [comment]
I added the vignetting effect directly to the master branch, because there were too many conflicts with the new code (new saturation, Gustavo's refactoring etc..). I am closing the pull request. Thanks

**Pull request closed without merging**